package.json

```diff
@@ -20,7 +20,9 @@
     "test-unit": "jest",
     "build:js-dev": "webpack --mode development",
     "build:js": "npm run lint && npm run format && webpack --mode production",
-    "build:py": "node ./extract-meta src/lib/components > dash_bio/metadata.json && python get_version_info.py && python -c \"import dash; dash.development.component_loader.generate_classes('dash_bio', 'dash_bio/metadata.json')\"",
+    "build:py": "dash-generate-components ./src/lib/components dash_bio -p package-info.json",
+    "build:py_and_r": "dash-generate-components ./src/lib/components dash_bio -p package-info.json && dash-generate-components ./src/lib/components dash_bio -p package-info.json --r-prefix='dashbio' --r-depends='manhattanly'",
+    "build:all": "npm run build:js && npm run build:js-dev && npm run build:py_and_r",
```

Comment:
Same suggestion as for DAQ
https://github.com/plotly/dash-daq/pull/79/files#r360213585

Reply:
fixed in fa9eba4, f1b7c3f